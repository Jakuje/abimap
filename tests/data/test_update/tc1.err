[WARNING] Overwriting existing file 'tc1.map'
[WARNING] Given paths in '--out' and 'file' are the same.
[WARNING] Moving 'tc1.map' to 'tc1.map.old'.
