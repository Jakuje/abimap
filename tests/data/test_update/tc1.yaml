# Testcases to test new command
-
  input:
    args:
      - "update"
      - "--add"
      - "--out"
      - "tc1.map"
      - "tc1.map"
    stdin: "tc1.in"
  output:
    file: "tc1.out"
    stdout: "tc1.stdout"
    stderr: "tc1.err"
