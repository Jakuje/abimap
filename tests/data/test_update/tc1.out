# This map file was automatically updated

LIBTC1_1_1_0
{
    global:
        new_symbol;
} LIBTC1_1_0_0;

LIBTC1_1_0_0
{
    global:
        another_symbol;
        symbol;
    local:
        *;
} ;

